admin/client.py

```diff
@@ -297,10 +297,19 @@ def main(reactor, args, base_path, top_level):
 
     try:
         nodes = yield runner.start_nodes(reactor, node_count=1)
-        yield perform(
-            make_dispatcher(reactor),
-            install_cli(runner.package_source, nodes[0]))
-        result = yield run_client_tests(reactor=reactor, node=nodes[0])
+        if options['pip']:
+            yield perform(
+                make_dispatcher(reactor),
+                install_cli_pip(runner.package_source, nodes[0]))
+            result = yield run_client_tests(
+                reactor=reactor, node=nodes[0], commands=task_cli_pip_test())
+        else:
+            yield perform(
+                make_dispatcher(reactor),
+                install_cli(runner.package_source, nodes[0]))
```

Comment:
Now that there's more than one way of installing the CLI, we should rename `install_cli` to `install_cli_system_packages`.